DESCRIPTION

```diff
@@ -49,7 +49,7 @@ Description: The aim of the 'report' package is to bridge the gap between
     This package converts statistical models and data frames into textual
     reports suited for publication, ensuring standardization and quality
     in results reporting.
-License: GPL-3 | file LICENSE
+License: MIT + file LICENSE
```

Comment:
You can also include a file like this in the repo: https://github.com/easystats/see/blob/main/LICENSE

Just replace with the correct package name.

Reply:
I think I deleted it because I vaguely remember CRAN asking to remove those license files when submitting, or is that specific to GPL-3? Do you know what I’m talking about?

Reply:
CRAN will complain if there is `LICENSE.md`, which is why we add it to `.Rbuildignore.`

But if you write `MIT + file LICENSE`, then that means you are also including the `LICENSE` file in the package bundle. So CRAN is not going to complain about that file, or at least it hasn't happened to me, and I include this file in all my packages.

Reply:
E.g. https://github.com/IndrajeetPatil/ggstatsplot/blob/main/LICENSE

Btw, automated way to do this is using [`{usethis}`](https://usethis.r-lib.org/reference/licenses.html)

Reply:
Oh ok, great, thanks for clearing that up!

Reply:
Yup, that’s what I use